mods/default/nodes.lua

```diff
@@ -48,7 +48,8 @@ Soft / Non-Stone
 default:dirt
 default:dirt_with_grass
 default:dirt_with_grass_footsteps
-default:dirt_with_dry_grass
```

Comment:
Node must be kept to avoid unknown nodes.

Reply:
done